scripts/start-shared-test-validator.sh

```diff
@@ -23,6 +36,7 @@ mkdir -p $LOCK_DIR
       validator_pid=$!
       echo "Started test validator (PID $validator_pid)"
       wait
+      wait_for_rpc_server
```

Comment:
I don't quite remember how this works, but I think this line will never run, because the `wait` above it is waiting for the `flock` to be released.

Reply:
I was curious if for some reason `flock` was releasing but then the validator was still undergoing startup after `flock` detached from the process.